docs/src/content/docs/id/reference/configuration.mdx

```diff
@@ -346,6 +379,94 @@ starlight({
 });
 ```
 
+### `expressiveCode`
+
+**type:** `StarlightExpressiveCodeOptions | boolean`  
+**default:** `true`
+
+Starlight menggunakan [Kode Ekspresif](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) untuk merender blok kode dan menambahkan dukungan untuk menyorot bagian contoh kode, menambahkan nama file ke blok kode, dan banyak lagi.
+Lihat [panduan “Blok kode”](/id/guides/authoring-content/#code-blocks) untuk mempelajari cara menggunakan sintaksis Kode Ekspresif dalam konten Markdown dan MDX Anda.
+
+Anda dapat menggunakan salah satu [opsi konfigurasi Kode Ekspresif](https://github.com/expressive-code/expressive-code/blob/main/packages/astro-expressive-code/README.md#configuration) standar serta beberapa properti khusus Starlight, dengan mengaturnya dalam opsi `expressiveCode` Starlight.
+Misalnya, gunakan opsi `styleOverrides` Expressive Code untuk mengganti CSS default. Ini memungkinkan kustomisasi seperti memberi blok kode Anda sudut membulat:
+
+```js ins={2-4}
+starlight({
+	expressiveCode: {
+		styleOverrides: { borderRadius: '0.5rem' },
+	},
+});
+```
+
+Jika Anda ingin menonaktifkan Kode Ekspresif, gunakan `expressiveCode: false` dalam konfigurasi Starlight Anda:
+
+```js ins={2}
+starlight({
+	expressiveCode: false,
+});
+```
+
+Selain opsi Kode Ekspresif standar, Anda juga dapat mengatur properti khusus Starlight berikut dalam konfigurasi `expressiveCode` Anda untuk lebih menyesuaikan aturan tema untuk blok kode Anda:
+
+#### `themes`
+
+**type:** `Array<string | ThemeObject | ExpressiveCodeTheme>`  
+**default:** `['starlight-dark', 'starlight-light']`
+
+Mengatur tema yang digunakan untuk memberi gaya pada blok kode.
+Lihat [Dokumentasi Expressive Code `themes`](https://github.com/expressive-code/expressive-code/blob/main/packages/astro-expressive-code/README.md#themes) untuk detail format tema yang didukung.
+
+Starlight menggunakan varian gelap dan terang dari [tema Night Owl](https://github.com/sdras/night-owl-vscode-theme) Sarah Drasner secara default.
+
+Jika Anda menyediakan setidaknya satu tema gelap dan satu tema terang, Starlight akan secara otomatis menjaga tema blok kode aktif tetap sinkron dengan tema situs saat ini.
+Konfigurasikan aturan ini dengan opsi [`useStarlightDarkModeSwitch`](#usestarlightdarkmodeswitch).
+
+#### `useStarlightDarkModeSwitch`
+
+**type:** `boolean`  
+**default:** `true`
+
+Jika `true`, blok kode secara otomatis beralih antara tema terang dan gelap saat tema situs berubah.
+Jika `false`, Anda harus menambahkan CSS secara manual untuk menangani peralihan antara beberapa tema.
+
+:::catatan
```

Comment:
```suggestion
:::note
```

The type of asides should not be translated and should always be `note`, `tip`, `caution` or `danger`.

Reply:
Ah, i'm sorry. I'm not notice this was a code. I will fix it soon. Thank you.